presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/CassandraType.java

```diff
@@ -317,6 +338,9 @@ public static String getColumnValueForCql(Row row, int i, CassandraType cassandr
                     return row.getUUID(i).toString();
                 case TIMESTAMP:
                     return Long.toString(row.getTimestamp(i).getTime());
+                case DATE:
+                    return CassandraCqlUtils.quoteStringLiteral(
+                            LocalDate.fromDaysSinceEpoch(row.getDate(i).getDaysSinceEpoch()).toString());
```

Comment:
is `LocalDate.fromDaysSinceEpoch(row.getDate(i).getDaysSinceEpoch()` equivalent to `row.getDate(i)`? or is it a different type?